src/main/java/io/xstefank/wildfly/bot/model/WildFlyConfigFile.java

```diff
@@ -4,16 +4,29 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.regex.Pattern;
 
 public class WildFlyConfigFile {
 
     public WildFlyConfig wildfly;
 
     public static final class WildFlyConfig {
 
+        private static final String DEFAULT_PROJECT_KEY = "WFLY";
+
+        private Pattern projectPattern = Pattern.compile(String.format("\\[%s-\\d+]\\s+.*|%s-\\d+\\s+.*", DEFAULT_PROJECT_KEY, DEFAULT_PROJECT_KEY));
+
+        public void setProjectKey(String name) {
+            projectPattern = Pattern.compile(String.format("\\[%s-\\d+]\\s+.*|%s-\\d+\\s+.*", name, name));
+        }
+
         public List<WildFlyRule> rules;
```

Comment:
don't mix field and method order

Reply:
This is done because of Jackson serialization field from yaml file. I will annotate it to make it obvious.